spec/actions_specs/crashlytics_spec.rb

```diff
@@ -220,7 +221,8 @@
               "-notesPath './fastlane/spec/fixtures/fastfiles/Fastfile1'",
               "-groupAliases 'groups,123'",
               "-notifications NO",
-              "-debug NO"])
+              "-debug NO"
+            ])
```

Comment:
Indent the right bracket the same as the first position after the preceding left parenthesis.